sdk/core/src/CommonNetworkMember/LegacyNetworkMemberWithFactories.ts

```diff
@@ -6,13 +6,20 @@ import { withDidData } from '../shared/getDidData'
 import { getOptionsFromEnvironment, ParsedOptions } from '../shared/getOptionsFromEnvironment'
 import { ParametersValidator } from '../shared/ParametersValidator'
 import { randomBytes } from '../shared/randomBytes'
-import { validateUsername } from '../shared/validateUsername'
-import UserManagementService from '../services/UserManagementService'
+import { validateUsername, UserManagementService } from '@affinidi/user-management'
 import { StaticDependencies, ConstructorUserData, createKeyManagementService } from './BaseNetworkMember'
 import { LegacyNetworkMember } from './LegacyNetworkMember'
+import { DEFAULT_COGNITO_REGION } from '../_defaultConfig'
+import { KeyStorageApiService } from '@affinidi/internal-api-clients'
+import { extractSDKVersion } from '../_helpers'
 
 const createUserManagementService = ({ basicOptions, accessApiKey }: ParsedOptions) => {
-  return new UserManagementService({ ...basicOptions, accessApiKey })
+  const keyStorageApiService = new KeyStorageApiService({
+    keyStorageUrl: basicOptions.keyStorageUrl,
+    accessApiKey,
+    sdkVersion: extractSDKVersion(),
+  })
+  return new UserManagementService({ ...basicOptions, region: DEFAULT_COGNITO_REGION }, { keyStorageApiService })
```

Comment:
Looks like it's time to extract `createUserManagementService` into a separate module and avoid all that code duplication.